astroquery/tap/xmlparser/utils.py

```diff
@@ -0,0 +1,52 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+"""
+=============
+TAP plus
+=============
+
+@author: Juan Carlos Segovia
+@contact: juan.carlos.segovia@sciops.esa.int
+
+European Space Astronomy Centre (ESAC)
+European Space Agency (ESA)
+
+Created on 30 jun. 2016
+
+
+"""
+
+import sys
+import io
+from astropy.table import Table as APTable
+
+
+def util_create_string_from_buffer(buffer):
+    if sys.version_info < (3,0,0):
```

Comment:
we usually use ``six`` (imported from ``from astropy.extern import six``) for all py2/py3 variations, so here you could use ```
if six.PY2:
    ...
else:
    ...
```